crates/holochain_zome_types/src/zome.rs

```diff
@@ -132,6 +133,23 @@ impl From<CoordinatorZome> for CoordinatorZomeDef {
     }
 }
 
+/// A zome defined by Wasm bytecode
+
+#[derive(Serialize, Deserialize, Hash, Clone, Debug, PartialEq, Eq, PartialOrd, Ord)]
+#[cfg_attr(feature = "test_utils", derive(arbitrary::Arbitrary))]
+pub struct WasmZome {
```

Comment:
why is this in zome types? this crate is for types we need compiled into both the wasm host and guest, but this seems like something we only need on the host?

Reply:
Well it's been in this file for years, it just got shifted up a few lines in this PR. But yes, sounds like it should probably be moved to `holochain_types`

Reply:
@thedavidmeister I agree but there's a bunch of stuff that would have to be rearranged so I'll add some TODOs but want to leave this out of this PR

Reply:
@maackle np